dotCMS/src/main/java/com/dotcms/rest/api/v1/experiment/AnalyticEventType.java

```diff
@@ -0,0 +1,7 @@
+package com.dotcms.rest.api.v1.experiment;
+
+public enum AnalyticEventType {
```

Comment:
Rename to "AnalyticsEventType"

Reply:
you can include this enum into the Event, so you can do Event.Type.CLICK for instance